static_src/components/users_invite.jsx

```diff
@@ -65,55 +58,68 @@ export default class UsersInvite extends React.Component {
   }
 
   get errorMessage() {
-    const err = this.props.error;
-    if (!err) return undefined;
-    const message = err.contextualMessage;
-    if (err.message) {
-      return `${message}: ${err.message}.`;
+    const { error } = this.props;
+
+    if (!error) return '';
+
+    const message = error.contextualMessage;
+
+    if (error.message) {
+      return `${message}: ${error.message}.`;
     }
+
     return message;
   }
 
+  get entityType() {
+    return this.isOrganization ? 'organization' : 'space';
+  }
+
+  get isOrganization() {
+    return /org/.test(this.props.inviteEntityType);
```

Comment:
We could compare against the exact string that `inviteEntityType` is. I didn't use a const because we don't currently (I dont think?) have a consistent way that we are handling those in the app.